packages/datadog_flutter_plugin/example/android/app/build.gradle

```diff
@@ -31,10 +31,6 @@ if (flutterVersionName == null) {
     flutterVersionName = '1.0'
 }
 
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
-
 android {
     compileSdkVersion 33
```

Comment:
should we target API 34 everywhere? this is current Android API, while 33 is the previous one.

Reply:
Probably. I'll do a separate PR to update SDK versions to 34.